pkg/issuer/venafi/client/venaficlient_test.go

```diff
@@ -240,6 +242,29 @@ func TestConfigForIssuerT(t *testing.T) {
 			},
 			expectedErr: false,
 		},
+		"if TPP and secret returns user/pass/clientId, should return config with those credentials": {
+			iss: tppIssuer,
+			secretsLister: generateSecretLister(&corev1.Secret{
+				Data: map[string][]byte{
+					tppUsernameKey: []byte(username),
+					tppPasswordKey: []byte(password),
+					tppClientIdKey: []byte(clientId),
```

Comment:
imo we should also have a test to verify that if client id is not provided in the secret, it correctly defaults to `cert-manager.io` as before.

Reply:
Agree with you, i have updated one of the tests to check default `client-id` value when not supplied